sklearn/utils/tests/test_validation.py

```diff
@@ -1616,9 +1616,9 @@ def test_get_feature_names_pandas_with_ints_no_warning(names):
     pd = pytest.importorskip("pandas")
     X = pd.DataFrame([[1, 2], [4, 5], [5, 6]], columns=names)
 
-    with pytest.warns(None) as record:
+    with warnings.catch_warnings():
+        warnings.simplefilter("error", UserWarning)
```

Comment:
```suggestion
        warnings.simplefilter("error", FutureWarning)
```

This is checking for:

https://github.com/scikit-learn/scikit-learn/blob/59428da95751fa92d46687850932f8f6ab1b4f3d/sklearn/utils/validation.py#L1844-L1848